### PR TITLE
fullfill promises

### PR DESCRIPTION
**Pull request by alexcos20**

### CODE REVIEW

**Review by kremalicious — APPROVED** [review]

File: src/balancer/OceanPool.ts [hunk 1]
```diff
@@ -630,7 +631,7 @@ export class OceanPool extends Pool {
       this.logger.error('ERROR: OCEAN approve failed')
       return null
     }
-    return this.swapExactAmountIn(
+    const tx = this.swapExactAmountIn(
```
Comment (kremalicious) [inline src/balancer/OceanPool.ts:634]: interesting that `this.swapExactAmountIn` does not need `await` to work in my local test

Comment (alexcos20) [reply src/balancer/OceanPool.ts:634]: it should work, but never the less..

Comment (kremalicious) [reply src/balancer/OceanPool.ts:634]: but typings say they are promises, so they should get an `await`

Comment (alexcos20) [reply src/balancer/OceanPool.ts:634]: fun fact is that swapExactAmountIn returns a promise, so as long as you do await BuyDT, you are resolving the promise

Comment (mihaisc) [reply src/balancer/OceanPool.ts:634]: same here, i can sell a dt, whilst on prod i can't

**Pull request merged**